### PR TITLE
Ui improvement

### PR DESCRIPTION
**Pull request by DoroNahari**

### CODE REVIEW

File: frontend/src/components/QuestionTypes/QuestionCheckbox.vue [hunk 1]
```diff
@@ -2,11 +2,9 @@
   <div>
     <p class="question-label">{{currentQuestion.message}}</p>
       <div v-for="(option,index) in options" :key="options[index].text">
-        <v-checkbox dense :label="option.text || option" v-model="option.selected"></v-checkbox>
+        <v-checkbox 
+      :error-messages="isValid" :hide-details="index !== (options.length - 1)" dense :label="option.text || option" v-model="option.selected"></v-checkbox>
       </div>
```
Comment (tomer-epstein) [inline frontend/src/components/QuestionTypes/QuestionCheckbox.vue]: isValidMessage

File: frontend/src/components/QuestionTypes/QuestionCheckbox.vue [hunk 2]
```diff
@@ -28,10 +26,14 @@ export default {
...
       return [];
+    },
+    isValid() {
+         return this.isAnswerValid(this.currentQuestion)
     }
   },
   props: {
-    currentQuestion: Object
+    currentQuestion: Object,
+    isAnswerValid: Function
```
Comment (tomer-epstein) [inline frontend/src/components/QuestionTypes/QuestionCheckbox.vue]: isValidMessage: Object

File: frontend/src/components/QuestionTypes/QuestionCheckbox.vue [hunk 3]
```diff
@@ -28,10 +26,14 @@ export default {
       }
 
       return [];
+    },
+    isValid() {
+         return this.isAnswerValid(this.currentQuestion)
```
Comment (tomer-epstein) [inline frontend/src/components/QuestionTypes/QuestionCheckbox.vue]: isValidMessage

File: frontend/src/assets/css/globalStyles.css [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+  font-size: var(--vscode-font-size);
+}
+/* Align v-card image to fit card */
+.v-card .v-image__image--cover {
+    background-size: contain;
+}
+.v-application button.v-btn:not(.v-btn--flat):not(.v-btn--text):not(.v-btn--outlined){
+  background-color: var(--vscode-button-background, #0e639c);
+  border-color: var(--vscode-button-background, #0e639c);
+  color: #ffffff !important;
```
[the remark this answers is not in the document]
Comment (DoroNahari) [reply frontend/src/assets/css/globalStyles.css]: Done

File: frontend/src/assets/css/globalStyles.css [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+  color: var(--vscode-input-foreground, #cccccc) !important;
+}
+/* Override error state colors on input components  */
+div.v-messages__message {
+  color: #ff5252 !important;
+}
+div.v-application .error--text {
+  color: var(--vscode-input-foreground, #cccccc) !important;
+  caret-color: var(--vscode-input-foreground, #cccccc) !important;
+}
```
[the remark this answers is not in the document]
Comment (DoroNahari) [reply frontend/src/assets/css/globalStyles.css]: I removed more.

File: frontend/src/components/QuestionTypes/QuestionInput.vue [hunk 6]
```diff
@@ -9,6 +9,7 @@
       aria-describedby="validation-message"
       solo
       dense
+      :error-messages="isValid"
```
[the remark this answers is not in the document]
Comment (DoroNahari) [reply frontend/src/components/QuestionTypes/QuestionInput.vue]: I converted all component's error-message to be like that again

**Comment by tomer-epstein** [comment]
found another bug ->this list is not shown.
image

**Comment by tomer-epstein** [comment]
selecting generator card hides the generator image.
image

**Comment by tomer-epstein** [comment]
check/uncheck one is checking all chices.
image

**Comment by DoroNahari** [comment]
> check/uncheck one is checking all chices.
> image

Fixed

**Pull request merged**